packages/lesswrong/components/form-components/ImageUpload2.tsx

```diff
@@ -0,0 +1,177 @@
+  socialPreviewImageId: {
+    width: 306,
+    height: 160
+  },
+}
+
+const ImageUpload2 = ({name, document, updateCurrentValues, clearField, label, croppingAspectRatio, placeholderUrl, classes}: {
+  name: string,
+  document: Object,
+  updateCurrentValues: Function,
+  clearField: Function,
+  label: string,
+  croppingAspectRatio?: number,
+  placeholderUrl?: string,
+  classes: ClassesType
+}) => {
+  const theme = useTheme();
+
+  const setImageInfo = (error, result) => {
+    if (error) {
+      throw new Error(error.statusText)
+    }
+    // currently we ignore all events other than a successful upload -
+    // see list here: https://cloudinary.com/documentation/upload_widget_reference#events
+    if (result.event !== 'success') {
+      return
+    }
+    const imageInfo = result.info
+    if (imageInfo && imageInfo.public_id) {
+      setImageId(imageInfo.public_id)
+      updateCurrentValues({[name]: imageInfo.public_id})
+    } else {
+      //eslint-disable-next-line no-console
+      console.error("Image Upload failed");
+    }
+  }
+
+  const uploadWidget = () => {
+    const cloudinaryArgs = cloudinaryArgsByImageType[name]
+    if (!cloudinaryArgs) throw new Error("Unsupported image upload type")
+    // @ts-ignore
+    cloudinary.openUploadWidget({
+      multiple: false,
+      sources: ['local', 'url', 'camera', 'facebook', 'instagram', 'google_drive'],
+      cropping: true,
+      cloudName: cloudinaryCloudNameSetting.get(),
+      theme: 'minimal',
+      croppingValidateDimensions: true,
+      croppingShowDimensions: true,
+      styles: {
+        palette: {
+            tabIcon: theme.palette.primary.main,
+            link: theme.palette.primary.main,
+            action: theme.palette.primary.main,
+            textDark: "#212121",
+        },
+        fonts: {
+            default: null,
+            "'Merriweather', serif": {
+                url: "https://fonts.googleapis.com/css?family=Merriweather",
+                active: true
+            }
+        }
+      },
+      ...cloudinaryArgs,
+      ...(croppingAspectRatio ? {croppingAspectRatio} : {})
+    }, setImageInfo);
+  }
+  
+  const removeImg = () => {
+    clearField()
+    setImageId(null)
+  }
+
+  const [imageId, setImageId] = useState(() => {
+    if (document && document[name]) {
+      return document[name];
+    }
+    return ''
+  })
+  
+  const formPreviewSize = formPreviewSizeByImageType[name]
+  if (!formPreviewSize) throw new Error("Unsupported image upload type")
+  
+  const imageUrl = imageId ? makeCloudinaryImageUrl(imageId, {
+    c: "fill",
+    dpr: "auto",
+    q: "auto",
+    f: "auto",
+    g: "auto:faces"
+  }) : placeholderUrl
+  
+  return (
+    <div className={classes.root} {...formPreviewSize}>
+      <Helmet>
+        <script src="https://upload-widget.cloudinary.com/global/all.js" type="text/javascript" />
+      </Helmet>
+      <div className={classes.imageBackground} style={{ backgroundImage: `url(${imageUrl})` }}>
```

Comment:
I think this should not try to apply a background image if `imageUrl` is not defined. Also maybe we should have some default background (such as a color or pattern) instead of no background? I'm not sure, I would ask Agnes for a suggestion.

<img width="654" alt="Screen Shot 2023-01-27 at 3 05 34 PM" src="https://user-images.githubusercontent.com/9057804/215187216-b4eff5b3-2cf0-4a6c-b77f-37c2c84d70d7.png">

Reply:
I have just made it fall back to a plain background for now (@agnesstenlund do you think this is ok?)
![Screenshot 2023-01-30 at 15 56 44](https://user-images.githubusercontent.com/77623106/215527380-3c57fdef-86f8-47b2-9850-f47811eddde9.png)

Reply:
yup I think white works well here 👍 I guess the truth is it would be the logo if it's empty, but this is likely more clear